src/otx/engine/engine.py

```diff
@@ -528,14 +528,16 @@ def explain(
         checkpoint: PathLike | None = None,
         datamodule: EVAL_DATALOADERS | OTXDataModule | None = None,
         explain_config: ExplainConfig | None = None,
+        dump: bool | None = False,
```

Comment:
why we need `None` Type for `dump`?

Reply:
This is a bug, will fix it in https://github.com/openvinotoolkit/training_extensions/pull/3014